### ISSUE TITLE
Notebook icon: Run in Section

### ISSUE THREAD

**Issue by Yoyokrazy** [microsoft/vscode-codicons#238]
**Is your icon request related to a problem? Please describe.**
- We are introducing new functionality to notebooks for Run Cells in Section, and Fold Section. This will come with a toolbar in the outline view, allowing users to execute and fold cells from that view pane.

**Describe the solution you'd like**
- A new icon for Run in Section, clearly distinguishing it from "Execute cell and below" and "Execute Cell and dependencies".

**Describe alternatives you've considered**
- Not rendering an icon as the primary action, and keeping both actions in the secondary overflow menu (decreasing discoverability, though there will still be the dots showing up which users haven't seen before)

**Additional context**
Picture of outline view in a notebook. The markdown header is focused with the primary action Run Cells in Section, and secondary action Fold Section. The outline entry for the code cell is being hovered over.
<img width="867" alt="image" src="https://github.com/microsoft/vscode-codicons/assets/12552271/1a3a6a79-0bb2-4a9e-abd0-519ea85ddab5">

Hover description:
<img width="378" alt="image" src="https://github.com/microsoft/vscode-codicons/assets/12552271/01c07146-4af7-4bcb-a472-34881944fa19">

Execute Cell and Below:
<img width="517" alt="image" src="https://github.com/microsoft/vscode-codicons/assets/12552271/040cc52c-fd43-4d76-b041-5acd213795eb">

Run with Dependencies codicon:
<img width="73" alt="image" src="https://github.com/microsoft/vscode-codicons/assets/12552271/e8219fa8-6f33-4f67-b4a4-a99033da78c0">

Potential thought for new icon. In my mind, sections/groupings are often designated with indentation, and they are in the outline pane specifically. 
<img width="54" alt="image" src="https://github.com/microsoft/vscode-codicons/assets/12552271/58de43c2-f965-4803-b284-97e8603cc3d0">

**Comment by hbons** [comment]
For folding we have `map` and `map-filled`:

<img width="481" alt="image" src="https://github.com/microsoft/vscode-codicons/assets/103326/55065ed8-ee78-4660-bee4-b6e50a1e5e2c">

I'm also adding a rotated version for cases where that makes more conceptual sense.

**Comment by hbons** [comment]
Instead of introducing a new concept, how about using the `run-all` icon? This makes in the group headers of the Outline view. Is this the only place the action will appear?
<img width="174" alt="image" src="https://github.com/microsoft/vscode-codicons/assets/103326/63b0650a-e5a9-4f69-b4bb-5a32d355fa32">

**Comment by daviddossett** [comment]
At a glance, I think the vanilla `run` icon is actually fine here since it's placed on the section header. Not sure a variant will really move the needle. We could try that first and see if it needs a more specific icon?

**Comment by mrleemurray** [comment]
@Yoyokrazy is this still required?

**Comment by Yoyokrazy** [comment]
Nope, nobody has spoken up about the standard run icon being insufficient. Closing as not planned 👍
